eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/boot/EventMeshAdminServer.java

```diff
@@ -120,160 +108,25 @@ public void parseHttpRequest(ChannelHandlerContext ctx, HttpRequest httpRequest)
         Optional<HttpHandler> httpHandlerOpt = adminHandlerManager.getHttpHandler(uri.getPath());
         if (httpHandlerOpt.isPresent()) {
             try {
-                AdminHttpExchange adminHttpExchange = new AdminHttpExchange(ctx, httpRequest);
-                httpHandlerOpt.get().handle(adminHttpExchange);
-                adminHttpExchange.writeAndFlash();
+                httpHandlerOpt.get().handle(httpRequest, ctx);
                 return;
             } catch (Exception e) {
-                log.error(e.getMessage(), e);
-                ctx.writeAndFlush(HttpResponseUtils.createInternalServerError()).addListener(ChannelFutureListener.CLOSE);
+                StringWriter writer = new StringWriter();
+                PrintWriter printWriter = new PrintWriter(writer);
+                e.printStackTrace(printWriter);
+                printWriter.flush();
+                String stackTrace = writer.toString();
+                Error error = new Error(e.toString(), stackTrace);
```

Comment:
`PrintWriter` is a legacy usage of log printing. It is recommanded to use Slf4j by now.